src/System.Windows.Extensions/src/System/Drawing/FontConverter.cs

```diff
@@ -126,94 +123,179 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
                 culture = CultureInfo.CurrentCulture;
             }
 
-            // Format is FontFamily, size[<units>[, style=1,2,3]]
-            // This is a bit tricky since the comma can be used for styles and fields
-            fields = font.Split(new char[] { culture.TextInfo.ListSeparator[0] });
-            if (fields.Length < 1)
+            char separator = culture.TextInfo.ListSeparator[0]; // For vi-VN: ','
+            string fontName = font; // start with the assumption that only the font name was provided.
+            string styleStr = null;
+            string sizeStr = null;
+            float fontSize = 8.25f;
+            FontStyle fontStyle = FontStyle.Regular;
+            GraphicsUnit units = GraphicsUnit.Point;
+
+            // Get the index of the first separator (would indicate the end of the name in the string).
+            int nameIndex = font.IndexOf(separator);
+
+            if (nameIndex < 0)
             {
-                throw new ArgumentException("Failed to parse font format");
+                return new Font(fontName, fontSize, fontStyle, units);
             }
 
-            font = fields[0];
-            f_size = 8f;
-            units = "px";
-            f_unit = GraphicsUnit.Pixel;
-            if (fields.Length > 1)
-            {   // We have a size
-                for (int i = 0; i < fields[1].Length; i++)
+            // Some parameters are provided in addition to name.
+            fontName = font.Substring(0, nameIndex);
+
+            if (nameIndex < font.Length - 1)
+            {
+                // Get the style index (if any). The size is a bit problematic because it can be formatted differently 
+                // depending on the culture, we'll parse it last.
+                int styleIndex = font.IndexOf(StyleHdr, StringComparison.CurrentCultureIgnoreCase);
```

Comment:
Previously this was invariant culture respecting case.
https://referencesource.microsoft.com/#System.Drawing/commonui/System/Drawing/FontConverter.cs,115
I assume this is an intentional change (fair enough)

Reply:
or possibly it should use `culture` (as it does for ConvertFromString below)

Reply:
oh I see below the test suggests it's intentional to be case insensitive. In that case, shouldn't it use `culture` which was pssed in, if any, rather than ignore it?
that would mean code like `culture.CompareInfo.IndexOf(font, StyleHdr, CompareOptions.IgnoreCase)`